content/applications/finance/accounting/taxes/retention.rst

```diff
@@ -2,69 +2,41 @@
 Withholding taxes
 =================
 
-A withholding tax, also called a retention tax, is a government
-requirement for the payer of a customer invoice to withhold or deduct
-tax from the payment, and pay that tax to the government. In most
-jurisdictions, withholding tax applies to employment income.
-
-With normal taxes, the tax is added to the subtotal to give you the
-total to pay. As opposed to normal taxes, withholding taxes are deducted
-from the amount to pay, as the tax will be paid by the customer.
-
-As, an example, in Colombia you may have the following invoice:
-
-.. image:: retention/retention03.png
-   :align: center
-
-In this example, the **company** who sent the invoice owes $20 of taxes to
-the **government** and the **customer** owes $10 of taxes to the **government**.
+A **withholding tax**, or retention tax, is a government requirement for the payer of a
+customer invoice to withhold or deduct tax from the payment and pay that tax to the government.
+Usually, the tax is added to the subtotal to determine the total amount to pay. In the case of
+withholding taxes, they are deducted from the payment amount.
```

Comment:
Suggestion to rephrase:
A withholding tax, also known as retention tax, mandates the payer of a customer invoice to deduct tax from the payment and remit it to the government. Typically, a tax is included in the subtotal to calculate the total amount paid, while withholding taxes are directly subtracted from the payment.